scripts/packaging/deb-maintainer-scripts/postinst

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -e
+
+action="$1"
+config_file="/etc/default/polkadot"
+
+if [ "$action" = "configure" ]; then
+  # Make user and group
+  getent group polkadot >/dev/null 2>&1 || addgroup --system polkadot
+  getent passwd polkadot >/dev/null 2>&1 ||
+    adduser --system --home /home/polkadot --disabled-password \
+    --ingroup polkadot polkadot
+  if [ ! -e "$config_file" ]; then
+    echo 'POLKADOT_CLI_ARGS=""' > /etc/default/polkadot
+  fi
```

Comment:
```suggestion
```

This isn’t actually needed, as systemd will treat an unset environment variable as an empty string.

Reply:
My intent was to populate the blank config file so that users can edit it as necessary without having to consult the README again to see what the name of the CLI arg environment variable was named.

Reply:
Good point.  That said, `/etc/default/polkadot` should be included in the package as a configuration file.